conf/default.yml

```diff
@@ -95,7 +95,7 @@ bitfield_field_reserved_prefix: 'Reserved'
 #     UINT64 Flags; //<<<<<<<<<<<<<
 #   }
 #
-bitfield_field_flags_name: 'Flags'
+bitfield_field_flags_name: 'AsUInt'
```

Comment:
Personally, "Flags" feels kinda better to me. Is there a deeper motivation for renaming this?

Reply:
The primary motivation was confusion I saw with my students, who had to read and write code using ia32.h. 

I _think_ `Flags` is less clearer to indicate it is an union of all fields, especially for those without Windows background.  

I am fine to revert it back to `Flags` if you like. I will be using my clone anyway.

Reply:
Alright, sounds fair. Let's use `AsUInt`.